CN instructs user to use syntax that doesn't exist (extract[verbose])

```
guso@000492-guso mission_protection_system % ./run_cn_proofs.sh
[1/1]: s_init
[1/5]: actuate_devices
[2/5]: c_NDEV
[3/5]: c_NVOTE_LOGIC
[4/5]: c_NTRIP
[5/5]: c_NINSTR
[1/7]: zero_u8_arr
instrumentation_common.c:54:9: warning: extract: index added, no resources (yet) extracted.
    (consider extract[verbose] or extract[quiet])
    /*@ extract Owned<uint8_t>, (u64)j;@*/
        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ 
[2/7]: zero_u32_arr
instrumentation_common.c:35:5: error: Missing resource for writing
    a[j] = 0;
    ~~~~~^~~ 
Resource needed: Block<unsigned int>(array_shift<unsigned int>(a, (u64)O_j0))
Consider the state in /var/folders/tz/g0kxkm2915xdtm6qgv8rff000000gp/T/state_617033.html
guso@000492-guso mission_protection_system % ./run_cn_proofs.sh
[1/1]: s_init
[1/5]: actuate_devices
[2/5]: c_NDEV
[3/5]: c_NVOTE_LOGIC
[4/5]: c_NTRIP
[5/5]: c_NINSTR
instrumentation_common.c:54:26: error: unexpected token after ']' and before 'Owned'
Please add error message for state 1074 to parsers/c/c_parser_error.messages
    /*@ extract[verbose] Owned<uint8_t>, (u64)j;@*/
    ```

There is mention of this in the code here https://github.com/rems-project/cerberus/blob/5fa5cf0f8d68d740b1994f111256b6d915a21835/backend/cn/check.ml#L1510 so I think it's a parsing issue.

--- a/backend/cn/check.ml
+++ b/backend/cn/check.ml
@@ -1501,19 +1501,13 @@ let rec check_expr labels (e : BT.t mu_expr) (k: IT.t -> unit m) : unit m =
                in
                let@ it = WellTyped.WIT.infer it in
                let@ (original_rs, _) = all_resources_tagged loc in
-               let verbose = List.exists (Id.is_str "verbose") attrs in
+               (* let verbose = List.exists (Id.is_str "verbose") attrs in *)
                let quiet = List.exists (Id.is_str "quiet") attrs in
-               (* if verbose *)
-               (* then Pp.print stdout (Pp.loc_headline loc (!^ "processing extract[verbose]")) *)
-               (* else (); *)
                let@ () = add_movable_index loc (predicate_name, it) in
                let@ (upd_rs, _) = all_resources_tagged loc in
-               let msg1 = "extract: index added, no resources (yet) extracted." in
-               let msg2 = "(consider extract[verbose] or extract[quiet])" in
                if (List.equal Int.equal (List.map snd original_rs) (List.map snd upd_rs)
                    && not quiet)
-               then warn loc (if verbose then (!^ msg1)
-                   else ((!^ msg1) ^^ Pp.hardline ^^ (!^ "    ") ^^ (!^ msg2)))
+               then warn loc (!^ "extract: index added, no resources (yet) extracted.")
                else ();
                return ()
             | M_CN_unfold (f, args) ->

